third_party/move/move-model/src/model.rs

```diff
@@ -3130,8 +3130,17 @@ impl<'env> ModuleEnv<'env> {
         if self.is_script_module() {
             return deps;
         }
+        // Since we need to check all inline functions, BFS is used
+        let mut reachable_funcs = VecDeque::new();
+        let mut visited = BTreeSet::new();
         for fun_env in self.get_functions() {
-            for used_fun in fun_env.get_used_functions().expect("used functions") {
+            reachable_funcs.push_back(fun_env.get_qualified_id());
+        }
+        while !reachable_funcs.is_empty() {
```

Comment:
We can use `while let Some(qfid) = reachable_funcs.pop_front() {..}`  instead. This avoid the `unwrap()` below, and we can use a single operation instead.